### PR TITLE
Introduce session_data tag support

### LINKED ISSUE #157: Add support for EXT-X-SESSION-DATA tag
`The EXT-X-SESSION-DATA tag allows arbitrary session data to be
   carried in a Master Playlist.`
https://tools.ietf.org/html/rfc8216#section-4.3.4.4

### PR DESCRIPTION
**Pull request by bvc3at**
Fix #157

### CODE REVIEW

File: m3u8/model.py [hunk 1]
```diff
@@ -913,6 +917,28 @@ def dumps(self):
...
+
+    def dumps(self):
+        session_data_out = ['DATA-ID=' + quoted(self.data_id)]
+
+        if self.value:
+            session_data_out.append('VALUE=' + quoted(self.value))
+        elif self.uri:
+            session_data_out.append('URI=' + quoted(self.uri))
+        if self.language:
+            session_data_out.append('LANGUAGE' + quoted(self.language))
```
Comment (davemevans) [inline m3u8/model.py]: Missing `=`?

Comment (bvc3at) [reply m3u8/model.py]: You're right, fix in dff8981

**Comment by leandromoreira** [comment]
can you add some tests for it?

**Comment by bvc3at** [comment]
> can you add some tests for it?

Sure thing, tests in 5d24363

**Pull request merged**